### PR TITLE
Adding code of conduct

### PR DESCRIPTION
**Pull request by KatherineMichel**
I think this works ok.

### CODE REVIEW

**Comment by williln** [comment]
LGTM but I'd like to defer to @jefftriplett jic! 

Thanks so much for getting on this so quickly, @KatherineMichel! You rock!

**Comment by KatherineMichel** [comment]
Happy to do it. Would be good for @jefftriplett to verify. When I tried to run it locally, what should have been index gave me a page with a directory structure instead. Assuming this will change when more permanent files are pushed to repo.

File: coc.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+## Photos
+
+In order to make DjangoCon US 2017 a great experience for everyone, do not photograph, video, or audio record anyone at DjangoCon without their express permission, sought in advance. If someone does not want to be photographed, video or audio recorded, please respect their wishes.
+
+Crowd shots are permitted, but when only the faces of a few people are visible, permission should be sought from all of those individuals.
+
+## License
+
+This Code of Conduct was forked from the example policy from the <a href="http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Policy">Geek Feminism wiki</a>, created by the Ada Initiative and other volunteers. which is under a Creative Commons Zero license.
+<p>Conference Code of Conduct by <a href="https://2016.djangocon.us/coc/">2016.djangocon.us/coc/</a> is licensed under a <a href="http://creativecommons.org/licenses/by/3.0/">Creative Commons Attribution 3.0 Unported License</a>.</p>
```
Comment (craigbruce) [inline coc.md]: This should be 2017 instead of 2016

**Review by craigbruce — COMMENTED** [review]
Minor date change

File: coc.md (hunk 1 above)
Comment (KatherineMichel) [reply coc.md]: Good catch. Thank you. I will fix it.

**Comment by jefftriplett** [comment]
LGTM! 🎉

**Pull request merged**